api/turing/validation/validator.go

```diff
@@ -42,9 +43,25 @@ func NewValidator(expSvc service.ExperimentsService) (*validator.Validate, error
 		return nil
 	}, router.RuleConditionOperator{})
 
+	err := instance.RegisterValidation("dns", validateDNSCompliantName)
+	if err != nil {
+		return nil, err
+	}
+
 	return instance, nil
 }
 
+func validateDNSCompliantName(fl validator.FieldLevel) bool {
```

Comment:
I'm sorry that I haven't pointed on it before, but can we just use `hostname_rfc1123` that's available in `go-playground/validator` straight out of the box? (https://github.com/go-playground/validator#network)

Reply:
!!! ok i'll change it to that.